config.json

```diff
@@ -824,6 +824,17 @@
       ]
     },
     {
+      "uuid": "7f4d5743-7ab8-42ca-b393-767f7e9a4e97",
+      "slug": "complex-numbers",
+      "core": false,
+      "unlocked_by": "leap",
+      "difficulty": 6,
```

Comment:
The same for difficulty.
What affected the score to be 6?

Reply:
Yeah, same hear. Took it from C# track. As I understood, difficulty grade is kinda... subjective.

Reply:
Agreed. Maybe we can come with some sort of formula to compute it, what do you think?

Reply:
I think this question maybe hurried somewhere in general discussions repo. Btw, will difficulty be used at all in version 2, or there would be relying on "unlocked_by" prerequisites?

In general, it could be done, of course, simplest heuristic would be to split topics that we have in to tiers, from simplest to hardest and add the difficulty level for each additional topic for exercise. So if basically tier_level1+....tier_level2

Reply:
@lilislilit, hm, this a really good point, thanks! I haven't thought about it this way
I've created the following issue https://github.com/exercism/discussions/issues/193 in general discussion, feel free to add your thoughts there